src/components/SelectedItemsPane.js

```diff
@@ -66,12 +66,16 @@ export const SelectedItems = ({ edit }) => {
         if (insideClass(e.target, "item")) return;
         revealForMe();
       }
+      if (e.key === "r") {
+        if (insideClass(e.target, "item")) return;
+        remove();
```

Comment:
I didn't add the confirm on purpose, as it can be really annoying when installing a new setup that varies a lot from the previous one.
We should either at some point:
- Allow this feature only for master,
- Add the confirmation here.

I would also only add this shortcut to the Help section only when one or the above (or the history do/undo) is implemented.